src/models/mods.rs

```diff
@@ -1,5 +1,71 @@
+use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 
+// TODO: use serde to parse this as base62
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct ModId(pub String);
+
+// TODO: use serde to parse this as base62
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct UserId(String);
+
+// TODO: what format should this be?
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct VersionId(String);
+
+// TODO: permissions, role names, etc
+#[derive(Serialize, Deserialize)]
+pub struct Team {
+    users: Vec<(String, UserId)>,
+}
+
```

Comment:
Move this struct to a different file, and simply reference a team ID here?

Reply:
Yeah, and I should also move the Id structs to their own file as well.